api/_globals/createImageBitmap.json

```diff
@@ -247,7 +247,7 @@
             "webview_android": "mirror"
           },
           "status": {
-            "experimental": false,
```

Comment:
This is actually in Safari, not sure which version though.
https://github.com/WebKit/WebKit/blob/main/Source/WebCore/html/ImageBitmapOptions.idl#L37

But that makes it non-experimental.